src/cfml/system/BaseCommand.cfc

```diff
@@ -252,10 +252,11 @@ component accessors="true" singleton {
 	/**
 	 * This will open a URL in the user's browser
  	 **/
-	function openURL( theURL ) {
+	function openURL( theURL, theBROWSER = "" ) {
```

Comment:
This does not follow casting guidelines.  URL is all uppercase because it is an acronym.  Browser is just a word. so headless camel case would give us `theBrowser` but the word "the" is also unnecessary.  I would just use `browser` or `defaultBrowser`  Though to be honest, I'm not sure this parameter should exist at all.  The default browser setting should simply be used internally and automatically in the open browser functionality.